src/ClusterBootstrap/scripts/setup_nfs_server.sh

```diff
@@ -1,10 +1,31 @@
+
+sudo mkdir -p /data
+sudo mount /data
+
+# setup NFS service
 sudo apt-get update
 sudo apt-get install -y nfs-kernel-server
 
-sudo mkdir -p /data/share 
+
+sudo mkdir -p /data/share
+sudo chmod -R 777 /data/share
 sudo chown nobody:nogroup /data/share
 
-echo "/data/share {{cnf["cloud_config"]["vnet_range"]}}(rw,sync,no_subtree_check,no_root_squash)" | sudo tee /etc/exports
-sudo systemctl restart nfs-kernel-server
+echo "/data/share 10.209.247.0/24(rw,fsid=1,nohide,insecure,sync,no_subtree_check,no_root_squash)" | sudo tee -a /etc/exports
+
+echo "/data/share 127.0.0.1/32(rw,fsid=1,nohide,insecure,sync,no_subtree_check,no_root_squash)" | sudo tee -a /etc/exports
+
+echo "/data/share 127.0.1.1/32(rw,fsid=1,nohide,insecure,sync,no_subtree_check,no_root_squash)" | sudo tee -a /etc/exports
+
+echo "/data/share 104.44.112.0/24(rw,fsid=1,nohide,insecure,sync,no_subtree_check,no_root_squash)" | sudo tee -a /etc/exports
```

Comment:
why hard coding here?

Reply:
this file is generated. it should not be tracked by git. would add it to gitignore